src/lando/api/legacy/workers/landing_worker.py

```diff
@@ -314,7 +314,7 @@ def run_job(self, job: LandingJob) -> bool:
                 # Try again, this is a temporary failure.
                 return False
             except Exception as e:
-                message = f"Unexpected error while fetching repo from {repo.pull_path}."
+                message = f"Unexpected error while fetching repo from {repo.name}."
```

Comment:
Why the switch to `repo.name` here? We don't overwrite the `pull_path`/`push_path` outside of `scm.push()`, right?

Reply:
We can also include the username and password in the push and pull paths. That's how suite does for CGit. It's a corner-case, but it's still a risk of leaking those (hardcoded) creds.